docs/governance/README.md

```diff
@@ -0,0 +1,127 @@
+# .NET Standard governance
+
+The .NET Standard represents the set of APIs that all .NET implementations have
+to provide. Over time, this list has to grow but it's critical that the set
+remains implementable by all current and future .NET implementations.
+
+To ensure that's the case, there is a formal process for proposing and accepting
+APIs into the .NET Standard. This document outlines the process.
+
+## Review board
+
+Today, we have several different .NET implementations:
+
+* .NET Core
+* .NET Framework
+* Mono
+* Unity
+* Xamarin
+
+Some of them have drastically different constraints, based on their runtime
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a review board that has to sign-off on API additions to the
+.NET Standard. The board comprises representatives from the following groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copies code from the .NET Framework &
+    .NET Core base class libraries, changes and extensions can impact their
+    ability to support the .NET Standard. Thus, we need to coordinate any
+    changes with Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+
+The chairman of the review board is [@migueldeicaza](https://github.com/migueldeicaza).
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
+
+## Process
+
+* **Anybody can submit proposals** for API additions to the .NET Standard.
+  Those will be tracked as issues and labeled with the [netstandard-api] label.
+* **We'll track features, no individual APIs**. In order to keep bookkeeping we
+  generally will usually file an issue per feature, rather than per API. The
+  issue description will then contain the APIs that should be added.
+* **New members on standardized types are automatically considered**. To prevent
```

Comment:
I think we need to forbid this until the “board agrees”. We can’t have types in the standard diverge differently as they make their way to the standard.

Reply:
Not sure I understand. Are you proposing that every time we add a member to a type that is in the standard the board has to agree? That isn’t a time efficient way to drive the platform.

I feel like the risk is extremely low. It’s extremely unlikely that we’ll have to reject members. And even if we do, nothing bad happens.

Reply:
I'm saying it can't just be considered then it has to just be "in".

Reply:
I think that is the default position.

Reply:
This would mean that all the span-based APIs on existing types are now automatically up for consideration.

Reply:
>This would mean that all the span-based APIs on existing types are now automatically up for consideration.

Yes the are.

I agree we don't want to block new API additions on existing types in a given .NET implementation as that would slow things down too much. However it is worth mentioning that if there are new APIs going in that might have issues that folks want feedback on they can take them through our API review process that we have setup http://aka.ms/apireview. While not a requirement it is a highly recommended for these type of APIs and probably worth mentioning in this doc.